.github/workflows/govmomi-build.yaml

```diff
@@ -46,17 +46,17 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.22
+          go-version: 1.23
```

Comment:
Have you consider simply refereceing `go.mod` to simplify the maintenance?

```yaml
      - name: Setup Go
        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
        with:
          go-version-file: "go.mod"
          cache: false
```

Note, this also shows the recommended practive to pin the to release commit hash.